src/main/java/com/notably/logic/commands/EditCommand.java

```diff
@@ -19,6 +19,11 @@
 import com.notably.commons.util.CollectionUtil;
 import com.notably.logic.commands.exceptions.CommandException;
 import com.notably.model.Model;
+import com.notably.model.person.Address;
+import com.notably.model.person.Email;
+import com.notably.model.person.Name;
+import com.notably.model.person.Person;
+import com.notably.model.person.Phone;
```

Comment:
Please remove all the above imports. These conflicted with the removal of these imports from #48 (merged PR)